### PR TITLE
PMM-5723 reviewdog check

### PR DESCRIPTION
**Pull request by BupycHuk**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -58,8 +60,15 @@ before_install:
...
   - go get -u github.com/golang/dep/cmd/dep
+  # install reviewdog and golangci-lin
+  - curl -sfL https://raw.githubusercontent.com/reviewdog/reviewdog/master/install.sh| sh -s
+  - curl -sSfL https://raw.githubusercontent.com/golangci/golangci-lint/master/install.sh | sh -s latest
 
 before_script:
+  # static analyze
+  - bin/golangci-lint run -c=.golangci-required.yml --out-format=line-number | bin/reviewdog -f=golangci-lint -level=error -reporter=github-pr-check
+  - bin/golangci-lint run -c=.golangci.yml --out-format=line-number | bin/reviewdog -f=golangci-lint -level=error -reporter=github-pr-review
+  - dep check
```
Comment (BupycHuk) [inline .travis.yml:71]: @percona-csalguero is it ok that dep check is failing?

Comment (percona-csalguero) [reply .travis.yml:71]: we don't upload vendor dir in PT

File: .travis.yml [hunk 2]
```diff
@@ -36,6 +36,8 @@ env:
     - MINIO_ENDPOINT: http://localhost:9000/
     - MINIO_ACCESS_KEY_ID: example00000
     - MINIO_SECRET_ACCESS_KEY: secret00000
+    - secure: "px8XYeNEAFTSTb1hYZuEOxqOXUxvp3EoU+KCtPck/KNozkoS95eBd9klgr3Os4wPKloLdMhrr0VE98lukogUxA/NmnYnos01kegjWgwwM6fkob8JxaN5KK4oUFF1wmirBlrjGlw8vUErPwINmrK4BywKpDbw6Yip6FzxdlWESHI="
```
Comment (percona-csalguero) [inline .travis.yml:39]: Minio is not really being used. We can remove it and remove this secure line

Comment (BupycHuk) [reply .travis.yml:39]: this secure is not for minio, it's for reviewdog. Will add comment

**Pull request merged**